mkdocs.yml

```diff
@@ -38,5 +38,5 @@ markdown_extensions:
   - pymdownx.details
   - pymdownx.superfences
 plugins:
-   - search
-   - awesome-nav
```

Comment:
Need to ensure awsome-nav is included in the plugin list, seems to have been deleted during merging. Best to rebase from main and squash this branch down.